infra/base-images/base-builder-go-codeintelligencetesting/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-go
+
+RUN install_go_CodeIntelligenceTesting.sh
+
```

Comment:
nit: Remove this line.

Reply:
Removed in [this commit](https://github.com/google/oss-fuzz/pull/7240/commits/19104a5462a4215b1ba6df269d59b6e31e2e5b7d).